### PR TITLE
🆕🎨 Misc styling III

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

Misc. styling updates:
- Addresses some items from https://github.com/cowprotocol/cowswap/issues/1408 

Screen Shot 2022-11-16 at 13 58 40
Screen Shot 2022-11-16 at 13 58 29

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @fairlighteth , great job!
I checked issues that are fixed in t #1408 (1, 2, 3, 5, 8, 9), and added 12 case there: 
image

Besides, I have a question related to the 9 case: is this OK that the exclamation sign icon is a white one in the dark mode? In the light mode the warning and the icon have the same color:
price updated exclamation

Thanks!

File: src/cow-react/modules/swap/pure/warnings.tsx [hunk 1]
```diff
@@ -73,8 +73,6 @@ export const NewSwapWarningsBottom = React.memo(function (props: NewSwapWarnings
           isSupportedWallet={isSupportedWallet}
         />
       )}
-
-      <NetworkAlert />
```
[the remark this answers is not in the document]
Comment (fairlighteth) [reply src/cow-react/modules/swap/pure/warnings.tsx:77]: To a different (inner) spot https://github.com/cowprotocol/cowswap/pull/1416/files#diff-6cdd19c6239200a099a1a6f28a4dd43cbf1158a4ae8d89f5ea815d5b4ce364de

**Review by alfetopito — APPROVED** [review]

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey @fairlighteth , I can see Swap button disabled in the PR when a warning is displayed in regular mode (no box to check it in the warning to enable Swap button)
image
Modes behavior is mixed up here: 
image

**Comment by fairlighteth** [comment]
@elena-zh Fixed it now

**Comment by elena-zh** [comment]
@fairlighteth , yes, thank you!
Could you please also enhance this warning or I need to open a separate issue for this? 
image

**Review by elena-zh — APPROVED** [review]
I have created #1432 issue to enhance Price impact warning

**Pull request merged**